builder_client.py

```diff
@@ -0,0 +1,43 @@
+import requests
+
+
+class BuilderClient(object):
+    """
+    Helper class for making requests to the Builder API which handles things like raising exceptions on invalid 
+    responses etc.
+    """
+    def __init__(self, api_key):
+        self.builder_session = requests.Session()
+        self.builder_session.headers.update({"Authorization": f"JWT {api_key}"})
+
+    def post(url, data=None, files=None, raise_error=True):
+        if data and files:
+            response = self.builder_session.post(url, data=data, files=files)
+        elif data:
+            response = self.builder_session.post(url, data=data)
+        else:
+            response = self.builder_session.post(url)
+
+        if raise_error:
+            response.raise_for_status()
+        return response
+
+    def patch(url, data=None, files=None):
+        if data and files:
+            response = self.builder_session.patch(url, data=data, files=files)
+        elif data:
+            response = self.builder_session.patch(url, data=data)
+        else:
+            response = self.builder_session.patch(url)
+
+        return response
+
+    def get(url):
+        response = self.builder_session.get(url)
+        response.raise_for_status()
```

Comment:
how come you're not providing the option to raise an error here?

Reply:
I only provided it as optional for `posts` because there was a specific instance that we wanted to do something different https://github.com/Guidebook/guidebook-zenefits-integration/blob/main/add_new_employee_webhook_receiver.py#L48.